google/cloud/functions/internal/call_user_function.cc

```diff
@@ -32,6 +33,33 @@ struct UnwrapResponse {
   }
 };
 
+namespace {
+BeastResponse ApplicationError(nlohmann::json const& error) {
+  auto msg = error.dump();
+  std::cerr << msg << std::endl;
```

Comment:
Debugging remnant?

Reply:
No, but certainly needed a comment explaining why. PTAL.

Reply:
I should have gotten that!  Thanks.